website/www/site/content/en/contribute/release-guide.md

```diff
@@ -398,15 +398,14 @@ There are 2 ways to perform this verification, either running automation script(
 Jenkins job `beam_Release_Gradle_Build` basically run `./gradlew build -PisRelease`.
 This only verifies that everything builds with unit tests passing.
 
-You can refer to [this script](https://gist.github.com/Ardagan/13e6031e8d1c9ebbd3029bf365c1a517) to mass-comment on PR.
+You can use [mass_comment.py](https://github.com/apache/beam/blob/master/release/src/main/scripts/mass_comment.py) to mass-comment on PR.
```

Comment:
Oh I see the website is already updated. We should remove the reference in line 391 as well.

Reply:
I missed that. Looks like it's mentioned here too:

https://github.com/apache/beam/blob/0a8d955e49776b872e8fda59cbeb0870afa68784/release/src/main/scripts/verify_release_build.sh#L32

Robin, can you make a PR and add me as reviewer?